packages/api-v2/src/AuthProvider.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { createContext, useState, useContext, useCallback, useEffect, useMemo } from 'react';
+import { useAPIContext } from './APIProvider';
+
+import { getActiveLoginIDFromLocalStorage, getToken } from '@deriv/utils';
+import useMutation from './useMutation';
+import { TSocketResponseData } from '../types';
+import { log } from 'console';
```

Comment:
minor: is this needed?

Reply:
I read about this that it is auto imported by vscode 🤔

Reply:
good point, its not, removed and pushed update, 
will have a look why its not linted (original /api package also does not seem have this linted)